khaya/asr_api.py

```diff
@@ -0,0 +1,26 @@
+from requests.models import Response
+
+from khaya.abstract_api import BASE_API
+
+
+class ASRAPI(BASE_API):
```

Comment:
Can we make this CapWords case?

Reply:
updates made. Ref commit `7afb86d34cf0e8ebc223824a8e383acbb3e72397`